### DISCUSSION TITLE
[Feature Request] Having a Color Blindness color scheme

### DISCUSSION THREAD

**Discussion by AmandaGBolton** [DannyDainton/newman-reporter-htmlextra#360] [Ideas]
**The default red/green color scheme is potentially challenging for people with the most common forms of color blindness.** Although all text would be readable, it would be difficult to quickly identify passing vs. failing tests.

**A third mode (vs. dark/light) that changes colors to ones that are more easily distinguished from each other could be helpful for these individuals (e.g., blue vs. red).** It is challenging to change it locally (for an organization) vs. using the default template, creating a barrier to gaining quick accessibility. It would also be more private if someone doesn't want to disclose their color blindness or raise the issue.

Adding symbols that quickly distinguish passed vs. failed tests could also be a helpful addition for all individuals.

**Comment by DannyDainton** [comment]
Hey @AmandaGBolton 

100% this!!

That's a great suggestion and I'd love to work with you on this!!

I can totally handle the implementation side of things but I'm honest going to struggle with getting the right things on the report.

Are you up for jumping on a call at some point and brainstorm some ideas?

**Comment by DannyDainton** [comment]
If you connect with me on LinkedIn / Twitter etc. we can have a chat about how to tackle this enhancement 🏆

**Comment by DannyDainton** [comment]
I was just checking out this site and there's a lot of much needed work to be done here. 

https://www.toptal.com/designers/colorfilter/

Protanopia:

![Screenshot_20200814_224749.jpg](https://user-images.githubusercontent.com/17089546/90295229-4a2f1080-de80-11ea-94ba-3cd78323e6ec.jpg)

**Comment by AmandaGBolton** [comment]
Hello! Wanted to leave a message here that I reached out to you on LinkedIn and I'm also happy to chat sometime.
